frontend/src/stores/levels/actions.ts

```diff
@@ -0,0 +1,21 @@
+import { levels } from './state'
+import { get } from '@/services/http'
+
+export async function fetchLevelsByRoleId(roleId: string) {
+  try {
```

Comment:
Try-catches should be in the component (Views), not in the actions. Please, put your error treatment in the View